code/game/gamemodes/cult/ritual.dm

```diff
@@ -56,22 +56,16 @@ It also contains rune words, which are soon to be removed.
 /obj/item/weapon/tome/examine(mob/user)
 	..()
 	if(iscultist(user))
-		user << "The scriptures of the Geometer. Allows the scribing of runes and access of knowledge archives."
+		user << "The scriptures of the Geometer. Allows the scribing of runes and access to the knowledge archives of the cult of Nar-Sie."
 
 /obj/item/weapon/tome/attack(mob/living/M, mob/living/user)
-	if(istype(M,/mob/dead/observer))
-		M.invisibility = 0
-		user.visible_message("<span class='warning'>[user] strikes the air with [src], and a spirit appears!</span>", \
-							 "<span class='danger'>You drag the ghost to your plane of reality!</span>")
-		add_logs(user, M, "smacked", src)
-		return

```

Comment:
You can use the Reveal Runes rune to make all nearby ghosts appear. This has been in the game for a while.